rtrlib/pfx_table.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf8 -*-
+"""
+rtrlib.rtr_manager
+------------------
+
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+
+from .exceptions import PFXException
+from .rtr_manager import ValidationResult
+from .util import ip_str_to_addr
+
+from _rtrlib import ffi, lib
+
+
+class PfxTable(object):
+    """
+    Wrapper class around pfx_table.
+
+    """
+
+    def __init__(self):
+        # allocate pfx_table
+        self.pfx_table = ffi.new('struct pfx_table *')
+        # initialize it
+        lib.pfx_table_init(self.pfx_table,
+                           ffi.NULL)
+        self.closed = False
+
+    @classmethod
```

Comment:
Please use `staticmethod` instead of `classmethod` here.
That way you do not need the `cls` parameter.

Reply:
Yes, you're right. It's more appropriate in this case. Done.